check_deps/check_python_dependencies.py

```diff
@@ -59,5 +59,5 @@
         print("\n\t\tRun apt-get install %s\n" % ims)
     sys.exit(1)
 
-print("All required modules installed")
+print("All required modules are installed")
```

Comment:
Sorry, everything ok with this change!

Reply:
Eh..? I was already in the process of writing an explanation...